### PR TITLE
Add size limit in admin

### PR DESCRIPTION
**Pull request by patjouk**
As I continue to work on optimizing the foundation site, I was surprised to see that images coming from pulse are well above 1MB.
For example, [the homepage](https://foundation.mozilla.org/en/) is 6MB, 4MB is images, 3.5MB **just** for pulse images.
Want something even scarier: https://www.mozillapulse.org/profile/82 is 61MB( image: 58MB), https://www.mozillapulse.org/latest is 14MB (images: 12MB)

Why 500kb? Because I was looking at the foundation site homepage and all the other images are below it and are still decent.

I'm adding this in admin to make sure that people login there to add entries won't be able to sneak something big again.

Similar PR is coming for pulse front.

### CODE REVIEW

**Comment by cadecairos** [comment]
In the review app admin, if I try to list profiles, I get a server timeout after 30 seconds - doesn't happen locally, or on production - @patjouk is this behaviour related to the PR somehow?

File: pulseapi/entries/forms.py [hunk 1]
```diff
@@ -39,3 +39,11 @@ def _save_m2m(self):
         for creator_profile_id in creator_profile_id_list:
             profile_id = int(creator_profile_id)
             EntryCreator.objects.create(entry=entry, profile_id=profile_id)
+
+    def clean_thumbnail(self):
+        data = self.cleaned_data["thumbnail"]
+        if not data:
+            return data
```
Comment (Pomax) [inline pulseapi/entries/forms.py]: might be cleaner to say `if data and data.size > ...` on the next line, instead?

File: pulseapi/entries/forms.py [hunk 2]
```diff
@@ -39,3 +39,11 @@ def _save_m2m(self):
         for creator_profile_id in creator_profile_id_list:
             profile_id = int(creator_profile_id)
             EntryCreator.objects.create(entry=entry, profile_id=profile_id)
+
+    def clean_thumbnail(self):
```
Comment (Pomax) [inline pulseapi/entries/forms.py]: As code used here as well as `pulseapi/profiles/forms.py`, lets put this in its own file in the utility dir, and the import it in both these forms.py files? 

E.g. something like:
```
from pulseapi.utility.form_helpers import disallow_large_images

...

    def clean_thumbnail(self):
        data = self.cleaned_data.get("thumbnail", None)
        return disallow_large_images(data) 
```

**Comment by patjouk** [comment]
@cadecairos it's unrelated to this PR. I opened another one for optimizing the queries for the profile pages in the admin.

**Pull request merged**